connectors/casa/main.py

```diff
@@ -28,6 +31,8 @@ def main(request):
     config = request.json.get("secrets", {})
     state = request.json.get("state", {})
 
+    logging.info(f"Received Fivetran request with state: {state}")
```

Comment:
So I'm not sure if this is still relevant, but I remember for some reason with logging you should use `%` formatting instead:

I quickly googled it and it seems like the library is optimised for `%` formatting:
https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs

But I have no strong opinions here.

Reply:
Yeah seems like the author found a specific case where f strings aren't preferred but I think just printing some state here should be fine (famous last words)

Reply:
Well, it won't break anything (even more famous last words... 🙈 ) so as I said your call. I'm marking this comment as resolved.